src/main/cpp/Robot.cpp

```diff
@@ -53,7 +57,8 @@ const void EncoderValues() {
     frc::SmartDashboard::PutNumber("Right Drive Voltage", Chassis::getInstance()->GetRightVoltage());
     frc::SmartDashboard::PutNumber("Roller current", CargoIntake::getInstance()->RollerCurrent());
     frc::SmartDashboard::PutNumber("HatchIntake Current", HatchIntake::getInstance()->GetCurrent());
-    frc::SmartDashboard::PutNumber("Climber Lift Limit Switch (reverse)", Climber::getInstance()->IsClimberUp());
+    frc::SmartDashboard::PutBoolean("Climber Lift Limit Switch (reverse)", Climber::getInstance()->IsClimberUp());
+    frc::SmartDashboard::PutBoolean("Climber Lift Limit Switch (other)", Climber::getInstance()->IsOtherLimitSwitch());
```

Comment:
Do we still want this `IsOtherLimitSwitch()` function since it was just used for testing?